ci/deps/ctms/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
```

Comment:
The absence of set -e here suggests that the script will just run unchecked regardless of errors. That's probably okay, in this case?